### PR TITLE
Realtime notifications for launches and projects

### PR DESCRIPTION
**Pull request by Jnegrier**
I spent a lot of time trying to follow a good component design for this, and also try to update the data in realtime.

In the end, nothing worked as good as intended, so this basic approximation could be just a baseline for a future better implementation.

I still left some comments on the other attempts, as the code could be fixed and implemented later.

### CODE REVIEW

File: components/templates/eventNotification.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      action={
+        <React.Fragment>
+          <Tooltip title="Reload page">
+          <Button color="secondary" size="small" onClick={handleReload}>
+          <ReplayIcon fontSize="small" />
+          </Button>
+          </Tooltip>
+        </React.Fragment>
+      }
+      key={vertical + horizontal}
```
Comment (opishcheiko) [inline components/templates/eventNotification.tsx:52]: fancy stuff, Juan!

File: pages/launches/[launchesByProjectId].tsx [hunk 2]
```diff
@@ -167,6 +220,25 @@ function Launches(props: Props) {
...
+              <Button color="secondary" size="small" onClick={handleReloadPage}>
+              <ReplayIcon fontSize="small" />
+              </Button>
+              </Tooltip>
+            </React.Fragment>
+          }
+          key={vertical + horizontal}
+        />
+        {/* Attempt to use notification as a component */}
+        {/* {notification? EventNotification("There are new launches 🚀") : EventNotification("Connecting for events...") }  */}
```
Comment (opishcheiko) [inline pages/launches/[launchesByProjectId].tsx:241]: I will have a look later if there is a way to use a component way

**Review by opishcheiko — APPROVED** [review]
Well done, big piece of work done! A start to a new era on delta

**Comment by Jnegrier** [comment]
> Well done, big piece of work done! A start to a new era on delta

Thanks Sasha!

**Pull request merged**